backends-common/elasticsearch-v6/src/main/java/org/apache/james/backends/es/v6/IndexCreationFactory.java

```diff
@@ -131,13 +131,18 @@ private XContentBuilder generateSetting(int nbShards, int nbReplica) throws IOEx
                         .field("number_of_shards", nbShards)
                         .field("number_of_replicas", nbReplica)
                         .startObject("analysis")
-                            .startObject("analyzer")
+                            .startObject("normalizer")
```

Comment:
can some one explain to me what this means?

Reply:
A normalizer here pretty much allows us too do some operations on keywords, that we need to use for sorting. In ES2, everything seemed to be mixed together. ES6 seems more strict by separating those different concepts and forcing you being more rigorous.

Official doc about normalizer : https://www.elastic.co/guide/en/elasticsearch/reference/6.7/normalizer.html

Reply:
The normalizer defined here allows us to do a case insensitive sort while doing our search